packages/grafana-ui/src/components/TimePicker/TimeOfDayPicker.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { FC } from 'react';
 import RcTimePicker from 'rc-time-picker';
+import classNames from 'classnames';
 import { dateTime, DateTime, dateTimeAsMoment } from '@grafana/data';
 
 interface Props {
   onChange: (value: DateTime) => void;
   value: DateTime;
   showHour?: boolean;
   minuteStep?: number;
+  width?: number;
 }
 
-export const TimeOfDayPicker: FC<Props> = ({ minuteStep = 1, showHour = true, onChange, value }) => {
+export const TimeOfDayPicker: FC<Props> = ({ minuteStep = 1, showHour = true, onChange, value, width }) => {
   return (
     <div>
       <RcTimePicker
+        className={classNames({ [`width-${width}`]: width })}
```

Comment:
@alexanderzobnin I guess it would be a good opportunity to use Emotion, but I'm not sure how to align the widths with the ones from the `scss` classes, so I added a className instead. 

Also not sure if that `div` wrapper is here for a reason or is it redundant, but I kept it so I don't break anything by accident.

Reply:
It's a good question. I'm trying to find examples in grafana-ui, but it seems, all of them use `classNames` as well. @dprokop do you have any thoughts on this, is it aligned with latest code style?

Reply:
Actually now that it has been decided not to make inputs fixed widths, this can be removed. However in future I think we'd still need a way to pass `width` as a prop to the input elements.

Reply:
We have standard sizes for Inputs

https://github.com/grafana/grafana/blob/165c3adf1d2e38c5cc01da6db02a0284e39b21f4/packages/grafana-ui/src/components/Forms/commonStyles.ts#L57

Reply:
@peterholmberg @alexanderzobnin I've replaced `width` with the `size` prop. I guess we'd look into doing the same for other input elements if we want to have unified styles/sizes.